pkg/operator/controller/ingress/deployment.go

```diff
@@ -287,6 +287,10 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 					},
 				},
 			},
+			// TODO: Once https://issues.redhat.com/browse/RFE-1759
```

Comment:
hm, I dont see us fixing this in 4.7 ever, but no harm in keeping the comments around I suppose.